config/database.yml

```diff
@@ -84,3 +86,11 @@ production:
   database: posts_apl_production
   username: posts_apl
   password: <%= ENV["POSTS_APL_DATABASE_PASSWORD"] %>
+
+  development:
```

Comment:
this section seem to be needless, or do you use `development.development` somewhere?